### DISCUSSION TITLE
Simulate realistic warming/cooling of oceans/lakes

### DISCUSSION THREAD

**Discussion by Weathermandavid** [niels747/2D-Weather-Sandbox#102] [Ideas]
It would be so cool if the ocean temperature or lake temperature was based on the air temperature based in real life. For an example the gulf of Mexico takes roughly 5-6 months for it to warm up from 25C to 33C due to higher sun angle and more solar radiational heating. but make it to where a cold front or some sort of temperature drop to make the water cool down gradually.

**Comment by Meso22** [comment]
Dynamic cooling and heating of water might be coming but I am not too sure
